CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add timing function variables
 - Add a `outline` variant to button
 - Create a `shadowed-box` mixin
+- Add a meta description default value from course introduction, 
+  blog excerpt, category description, person bio and program excerpt
```

Comment:
did you forget organization?

Reply:
No I didn't forget. I intentionally don't include the organizations on this development, because the organization page don't have any simple text placeholder that resume its page.

Reply:
True :thinking: So if you agree on the necessity for other obejcts to also format the text (strip html and trunk), we could use the content of the body placeholder and take the first few characters of it?

Reply:
Yes, I agree! This works. We don't event need any custom template filter.

```html
<meta name="description" content="{% filter striptags|slice:":200" %}{% show_placeholder 'description' current_page %}{% endfilter %}" />
```

I'm going to update the PR and include the max 200 characters on the tests.

Reply:
Awesome! The "Django is batteries included" claim is real :star_struck: 
About the 200, I checked further:
> The ideal meta description length is 155-160 symbols with spaces.

Reply:
Ok. So I've hard coded / sliced it to 160 chars.